### PR TITLE
✨Automagically expire hacktoberfest opportunities on November 2

### PR DESCRIPTION
**Pull request by vikaspotluri123**
no issue

I'm not sold on the committing logic, do you know of an easy way to test it? The committer email is based on [this thread](https://github.community/t/github-actions-bot-email-address/17204/6)

### CODE REVIEW

**Comment by vikaspotluri123** [comment]
@aslafy-z bump 😄

File: scripts/expire-hacktoberfest.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		opportunities,
+		// JSON.stringify will add spaces around an array, which is not how we store arrays in JSON, so
+		// we manually stringify the array. Since we're returning a string, the double quotes (") would
+		// be escaped, so we use a replacer (`!;!`) (see next comment)
+		(key, value) => key === 'tags' ? `[!;!${value.join('!;!, !;!')}!;!]` : value,
+		2
+	// Which is replaced after stringification. The final thing we have to do is "convert" the string
+	// to an array, which we do by removing the start and end quotes from the value
+	// (e.g. "[]" gets transformed to [])
+	).replace(/!;!/g, '"').replace(/(\s+"tags": )"(.*)"/gm, '$1$2');
```
Comment (aslafy-z) [inline scripts/expire-hacktoberfest.js:26]: I don't get this part. How does hacktoberfest opportunities get tagged as expired in this code?

Comment (vikaspotluri123) [reply scripts/expire-hacktoberfest.js:26]: It doesn't get done here, it get's done in the first part. This part is just serializing the JSON

Comment (aslafy-z) [reply scripts/expire-hacktoberfest.js:26]: Oops you're right, I missed the push...

**Review by aslafy-z — APPROVED** [review]
LGTM

**Comment by vikaspotluri123** [comment]
Worked flawlessly! https://github.com/swapagarwal/swag-for-dev/commit/ec0b87b13c4aab870c61cf8d63740572b3208678

**Comment by swapagarwal** [comment]
Noice! 🤟 😎

**Pull request merged**